app-modules/engagement/config/roles/web/engagement_management.php

```diff
@@ -8,5 +8,14 @@
         'engagement_deliverable' => [
             '*',
         ],
+        'engagement_batch' => [
+            '*',
+        ],
+        'engagement_response' => [
+            '*',
+        ],
+        'engagement_file' => [
+            '*',
+        ],
```

Comment:
The fact that I named this "EngagementFile" was kind of a mistake. This is a separate concept from Engagements and is actually just Document storage on an Educatable.

Let's move this to be managed on it's own.